testsuite/path.c

```diff
@@ -202,6 +203,8 @@ WRAP_2ARGS(int, -1, stat, struct stat*);
 WRAP_2ARGS(int, -1, lstat, struct stat*);
 WRAP_2ARGS(int, -1, stat64, struct stat64*);
 WRAP_2ARGS(int, -1, lstat64, struct stat64*);
+WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
```

Comment:
```suggestion
struct __stat64_t64;
extern int __stat64_time64 (const char *file, struct __stat64_t64 *buf);
extern int __lstat64_time64 (const char *file, struct __stat64_t64 *buf);
WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
```
(I did not find an easy way to get those prototypes from `sys/stat.h`. The `WRAP_2ARGS` is unmodified, I just add those three lines above.)